process/smartContract/process.go

```diff
@@ -894,6 +902,12 @@ func (sc *scProcessor) isSCExecutionAfterBuiltInFunc(
 		return false, nil, nil
 	}
 
+	callType := determineCallType(tx)
+	if callType == vmcommon.AsynchronousCallBack {
```

Comment:
Is this a case of backwards compatibility?

Reply:
yes it is. this code was not enabled yet.